.github/workflows/eamxx-gh-clang-format.yml

```diff
@@ -105,19 +105,44 @@ jobs:
                "execute \`git apply <patch-file>\` from your local E3SM " \
                "branch." >> "${GITHUB_STEP_SUMMARY}"
           echo "Then commit the changes and push to your remote branch." >> "${GITHUB_STEP_SUMMARY}"
+          echo "" >> "${GITHUB_STEP_SUMMARY}"
+          echo "For diffs with ***fewer than 10 files***, we also include the text of " \
+               "the patch file in a collapsed section, below."  >> "${GITHUB_STEP_SUMMARY}"
+          if [[ $(wc -l < diffiles.txt) -lt 10 ]]; then
+            echo "<details>" >> "${GITHUB_STEP_SUMMARY}"
+            echo "" >> "${GITHUB_STEP_SUMMARY}"
+            echo "<summary>" >> "${GITHUB_STEP_SUMMARY}"
+            echo "### Contents of \`clang-format-patch.diff\`" >> "${GITHUB_STEP_SUMMARY}"
+            echo "</summary>" >> "${GITHUB_STEP_SUMMARY}"
+            echo "To apply this version of the patch, save the text to a local file, " \
+                 "and \`git apply <patch-file>\`, etc., as above." >> "${GITHUB_STEP_SUMMARY}"
+            echo "" >> "${GITHUB_STEP_SUMMARY}"
+            echo "\`\`\`" >> "${GITHUB_STEP_SUMMARY}"
+            cat clang-format-patch.diff >> "${GITHUB_STEP_SUMMARY}"
+            echo "\`\`\`" >> "${GITHUB_STEP_SUMMARY}"
+            echo "" >> "${GITHUB_STEP_SUMMARY}"
+            echo "</details>" >> "${GITHUB_STEP_SUMMARY}"
+          fi
         else
           echo "status_fail=false" >> "${GITHUB_ENV}"
           echo "## Status: \`clang-format\` Check Passed!" > "${GITHUB_STEP_SUMMARY}"
         fi
     - name: diff format changes
-      run: git diff > "components/eamxx/clang-format-patch.diff"
+      run: |
```

Comment:
This is fine, but...why do we need a unique file name? Different runs artifacts can have the same name, as they are stored separately on gh. The only issue is if someone downloads multiple patches from multiple PRs, but that's quite the corner case...

Reply:
lol, I still have to finish to go through my unread notifications. I guess that's my next one. :)